Setup base React project

mrniket opened the pull request:
This PR adds a react project with an example to the game_frontend folder inside of this repo. It also includes documentation and examples of how to test each part of the project. Please have a look and comment on the architecture!


<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ocadotechnology/aimmo/500)
<!-- Reviewable:end -->

OlafSzmidt commented:
Reviewed 10 of 19 files at r1, 2 of 7 files at r2, 20 of 21 files at r3.
Review status: 32 of 33 files reviewed at latest revision, 2 unresolved discussions.

---

*[README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70DZ1mYtAEuouqVDuW:-L70DZ1mYtAEuouqVDuX:b-bw652w) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L19)):*
> ```Markdown
> ## The Game
> 
> **Note** the game frontend lives [here](https://github.com/ocadotechnology/aimmo/tree/react_integration/game_frontend)
> ```

`**Note**: The` would look better

---

*[game_frontend/src/components/Movie/index.js, line 20 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70F2SRCuJW0WxEox6Q:-L70F2SRCuJW0WxEox6R:bedn8k9) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Movie/index.js#L20)):*
> ```JavaScript
>     director: PropTypes.string.isRequired,
>     producer: PropTypes.string.isRequired,
>     release_date: PropTypes.string.isRequired,
> ```

Could we change this from a string?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Review status: 32 of 33 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 23 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70eOFWYlpEVDb48Oow:-L70eOFWYlpEVDb48Oox:b-2fk8fb) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L23)):*
> ```Markdown
> The world is a isometric 3D grid. Some cells are impassable. Some cells generate score. Some cells contain pick-ups.
> 
> There are other Avatars in the world. The more Avatars there are, the bigger the world gets.
> ```

Right now it's not working - we'll redo the map expansion in a following sprint

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Review status: 32 of 33 files reviewed at latest revision, 4 unresolved discussions.

---

*[game_frontend/src/components/Button/index.js, line 5 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70nDhxoVGOFFgqeAgm:-L70nDhxoVGOFFgqeAgn:b-ktdfhc) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Button/index.js#L5)):*
> ```JavaScript
> const Button = styled.button`
>   padding: 10px
>   background-color: #3886AA
> ```

is there a way to create custom colours with names, like "white" below but custom? like in sass

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Reviewed 1 of 21 files at r3.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Reviewed 1 of 21 files at r3.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Reviewed 2 of 19 files at r1, 1 of 21 files at r3.
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L70zzYvuonkW2u_4Pa3:blvx3xm) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
> ```
> node_modules
> ```

Why is it not in the normal .gitignore at the root of the project?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L7188y20V-nioAHTEnt:b-8fx2u7) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

Why is it not in the normal .gitignore at the root of the project?
</blockquote></details>

:+1: 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Reviewed 8 of 19 files at r1, 2 of 7 files at r2, 19 of 21 files at r3.
Review status: all files reviewed at latest revision, 19 unresolved discussions.

---

*[game_frontend/README.md, line 7 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714B5UX0-sNDq0ccXf:-L714B5UX0-sNDq0ccXg:bw2rv75) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L7)):*
> ```Markdown
> ## Description
> 
> This folder contains the frontend for the AI:MMO game. It is written in single page application written in [React](https://reactjs.org/). We use [Redux](https://redux.js.org/) for state management and [redux-observable](https://redux-observable.js.org/) for handling our side effects (e.g. asynchronous calls). To learn more about how we connect all of these together, feel free to check out our [How does it all work](#how-does-it-all-work) section.
> ```

It is written in single page application written in.... => rephrase?

---

*[game_frontend/README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714LNxHctgntZ-_kU0:-L714LNxHctgntZ-_kU1:b-bayrz4) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L19)):*
> ```Markdown
> ### Build dependencies
> 
> Once you have cloned this repository, open up your terminal and `cd` into this folder and run:
> ```

this is a UNIX specific instruction

---

*[game_frontend/README.md, line 61 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L7162EIyVjlqE7mJk_v:-L7162EIyVjlqE7mJk_w:b-f2d104) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L61)):*
> ```Markdown
> things:
> * It sets the initial state for each of the reducers (see reducers)
> * Configures the store using the global initial state mentioned above (see stores)
> ```

where are the link?

---

*[game_frontend/README.md, line 67 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714_trLdqebgIQ6EKO:-L714_trLdqebgIQ6EKP:b8fzb6) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L67)):*
> ```Markdown
> ### Containers
> As the JSX code is rendered, usually an initial container is specified
> there so let's dig deeper (see "a bit on containers vs components")
> ```

a link to an anchor might be better if possible?

---

*[game_frontend/README.md, line 77 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L71524AQfhPhkAOiMPt:-L71524AQfhPhkAOiMPu:bw2yqbc) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L77)):*
> ```Markdown
> 
> We may want to add some user interaction with the state here, for example some logic when
> a user presses a button. To do this particular example we can assign a action dispatch function
> ```

AN action dispatch...

---

*[game_frontend/README.md, line 78 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L7156iWABwN2r41_PhK:-L7156iWABwN2r41_PhL:b-iul730) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L78)):*
> ```Markdown
> We may want to add some user interaction with the state here, for example some logic when
> a user presses a button. To do this particular example we can assign a action dispatch function
> (connected to the component in the `connect` mentioned above by assigning it to a prop) to a `onClick`
> ```

AN onClick?

---

*[game_frontend/README.md, line 83 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L715EU6l1T1WKDuf2Dp:-L715EU6l1T1WKDuf2Dq:b4v9fgq) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L83)):*
> ```Markdown
> 
> ### Actions & Types
> Actions are part of the Redux architecture. They usually live in `redux/featutes/FeatureName/actions.js`.
> ```

features not featutes

---

*[game_frontend/README.md, line 105 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L715_ynA-5csQC4JYuF:-L715_ynA-5csQC4JYuG:b-8u18d1) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L105)):*
> ```Markdown
> 
> Actions that get dispatched usually pass through some kind of middleware. In our
> case this will `redux-observable` which provide **epics**. We can check which
> ```

this will BE?

---

*[game_frontend/README.md, line 113 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L715ir8RGq8DSagF2O6:-L715ir9IOzYPfGUF5_m:bh8t9jn) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L113)):*
> ```Markdown
> Arguably the most important piece of the whole architecture. This specifies how the
> application's **state changes** in response to payloads of informations received by
> **actions**. Reducers do not *mutate* the state, they create *new* state. Remember,
> ```

create A new state

---

*[game_frontend/README.md, line 128 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L716A1gbYScE4FLqm7V:-L716A1gbYScE4FLqm7W:b-2z7dse) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L128)):*
> ```Markdown
> `mapStateToProps` to the container; then our props change and the
> React DOM appropriately updates the DOM, thus showing a response from the
> page to user action.
> ```

clunky wording?

---

*[game_frontend/src/components/Movie/index.js, line 20 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70F2SRCuJW0WxEox6Q:-L717GGc6TiMo5PIbJ4B:b4q6c70) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Movie/index.js#L20)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

Could we change this from a string?
</blockquote></details>

yeah isn't there a better format for dates?

---

*[game_frontend/src/components/Movie/__snapshots__/index.test.js.snap, line 5 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L717bV5Km1CojvrdXto:-L717bV5Km1CojvrdXtp:b-88tk8s) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Movie/__snapshots__/index.test.js.snap#L5)):*
> ```
> exports[`<Movie /> renders correctly 1`] = `
> <li>
>   Test Title
> ```

colour code shows Test as a keyword Oo

---

*[game_frontend/src/containers/GamePage/index.test.js, line 32 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L718u6NnMh9MthT63uW:-L718u6NnMh9MthT63uX:b32viy) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/containers/GamePage/index.test.js#L32)):*
> ```JavaScript
>   })
> 
>   it('should call fetchMovies when I click on the button', () => {
> ```

:O

---

*[game_frontend/src/containers/GamePage/__snapshots__/index.test.js.snap, line 101 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L719Fq1DXjrMN3e7QYC:-L719Fq1DXjrMN3e7QYD:b-lbxvd0) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/containers/GamePage/__snapshots__/index.test.js.snap#L101)):*
> ```
>           "nodeType": "host",
>           "props": Object {
>             "children": "Welcome to the AIMMO game screen",
> ```

a lot of strings are repeated everywhere...

---

*[game_frontend/src/redux/middlewares/logger.js, line 0 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L711DQ766P2tuo37UUc:-L711DQ766P2tuo37UUd:b-kq223p) ([raw file](https://github.com/ocadotechnology/aimmo/blob/cc93aa0fad854678d579a910a90d44618ff6346c/game_frontend/src/redux/middlewares/logger.js#L0)):*
huh? it's empty

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/README.md, line 106 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L71ANgp7_YkFJe99IQ0:-L71ANgp7_YkFJe99IQ1:bnp7f6e) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L106)):*
> middlware
middleware

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714LNxHctgntZ-_kU0:-L74K3n9Fk242V5Y7mlk:b-efd6kd) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L19)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

this is a UNIX specific instruction
</blockquote></details>

Don't think it is, WINDOWS shell has the same command

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/src/components/Movie/index.js, line 20 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70F2SRCuJW0WxEox6Q:-L74KteQAAXyeIfpyKyo:bxewchl) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Movie/index.js#L20)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

yeah isn't there a better format for dates?
</blockquote></details>

Not that it makes a big difference anyway as this is an example, but would be good to know how we can handle these data sets

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714LNxHctgntZ-_kU0:-L752_UimSVkwrAOIa11:biejhzj) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L19)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

Don't think it is, WINDOWS shell has the same command
</blockquote></details>

no - it's CD in capital letters ;) 

---

*[game_frontend/README.md, line 106 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L71ANgp7_YkFJe99IQ0:-L752iA6w-LXp-cmTzpy:b4c9lyr) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L106)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

> middlware
middleware
</blockquote></details>

:thumbsup: 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

mrniket commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[README.md, line 23 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70eOFWYlpEVDb48Oow:-L75-jJ0zNrfhQyCbDlU:b-63hfe4) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L23)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

Right now it's not working - we'll redo the map expansion in a following sprint
</blockquote></details>

:thumbsup: I think I'm going to rewrite this section, I'll use the comments here to guide me

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L757lfYGHiLFb6Rs_Sg:bxadwyu) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

:+1: 
</blockquote></details>

I would argue that it makes more sense to put the things that are specific to the javascript project in the .gitignore here. See https://stackoverflow.com/questions/3305869/are-multiple-gitignores-frowned-on which I used to make this decision

---

*[game_frontend/README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714LNxHctgntZ-_kU0:-L759XsQEZYd8hxGRQ3r:b-z3ol9e) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L19)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

no - it's CD in capital letters ;) 
</blockquote></details>

I will remove the `cd` command and just say "go to this folder"

---

*[game_frontend/src/components/Button/index.js, line 5 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70nDhxoVGOFFgqeAgm:-L759lmyBJz44n28y1HK:bgaohkb) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Button/index.js#L5)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

is there a way to create custom colours with names, like "white" below but custom? like in sass
</blockquote></details>

Yes there is! You can provide a theme. See this for more info: https://www.styled-components.com/docs/advanced#theming

---

*[game_frontend/src/components/Movie/index.js, line 20 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70F2SRCuJW0WxEox6Q:-L759xR8U5wzusHZey7a:b-u8htej) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Movie/index.js#L20)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

Not that it makes a big difference anyway as this is an example, but would be good to know how we can handle these data sets
</blockquote></details>

I'm not sure if it's worth changing in this example code, but yes it's possible: https://stackoverflow.com/questions/39627545/react-prop-validation-for-date-objects

---

*[game_frontend/src/components/Movie/__snapshots__/index.test.js.snap, line 5 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L717bV5Km1CojvrdXto:-L75AQTz5acLu5SY6AcW:b4jobxb) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Movie/__snapshots__/index.test.js.snap#L5)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

colour code shows Test as a keyword Oo
</blockquote></details>

¯\_(ツ)_/¯

---

*[game_frontend/src/containers/GamePage/index.test.js, line 32 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L718u6NnMh9MthT63uW:-L75Aeq6X034U65tBhYF:b1t9yov) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/containers/GamePage/index.test.js#L32)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

:O
</blockquote></details>

like it? :)

---

*[game_frontend/src/containers/GamePage/__snapshots__/index.test.js.snap, line 101 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L719Fq1DXjrMN3e7QYC:-L75AwIhLcn8uZoeLaXo:b-jpntlf) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/containers/GamePage/__snapshots__/index.test.js.snap#L101)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

a lot of strings are repeated everywhere...
</blockquote></details>

this is a test snapshot which is generated by tests in index.test.js.snap, so I don't think this is an issue .

---

*[game_frontend/src/redux/middlewares/logger.js, line  at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L711DQ766P2tuo37UUc:-L75CnX9GzoucmX3NOQ5:b-6tba5w) ([raw file](https://github.com/ocadotechnology/aimmo/blob/cc93aa0fad854678d579a910a90d44618ff6346c/game_frontend/src/redux/middlewares/logger.js#L)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

huh? it's empty
</blockquote></details>

ah yes I forgot to write this, I don't think it's necessary for the example and in reality, you would use something like redux-dev-tools so I'm going to remove this file

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/src/containers/GamePage/__snapshots__/index.test.js.snap, line 101 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L719Fq1DXjrMN3e7QYC:-L75Ly2kAU6idyzUaZ-L:b-vswvwx) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/containers/GamePage/__snapshots__/index.test.js.snap#L101)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

this is a test snapshot which is generated by tests in index.test.js.snap, so I don't think this is an issue .
</blockquote></details>

Supposed to be like this, auto generated file 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L75MIKwvUh4cdQ7N_4T:bp4wck2) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

I would argue that it makes more sense to put the things that are specific to the javascript project in the .gitignore here. See https://stackoverflow.com/questions/3305869/are-multiple-gitignores-frowned-on which I used to make this decision
</blockquote></details>

Which of the two points apply to us from that answer? I think neither? This only makes it harder to find the relevant file. One global one would be much easier to maintain

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/src/components/Movie/index.js, line 20 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70F2SRCuJW0WxEox6Q:-L75MYvK9g_USFBAzm_X:b-ahle5t) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Movie/index.js#L20)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

I'm not sure if it's worth changing in this example code, but yes it's possible: https://stackoverflow.com/questions/39627545/react-prop-validation-for-date-objects
</blockquote></details>

Good to know!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L7A25GgecNBfC6B-Q8o:b3kt8ub) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

Which of the two points apply to us from that answer? I think neither? This only makes it harder to find the relevant file. One global one would be much easier to maintain
</blockquote></details>

Yup. I'd vote for a single .gitignore 

---

*[game_frontend/src/containers/GamePage/index.test.js, line 32 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L718u6NnMh9MthT63uW:-L7A2w2vofeMMeWZHs4P:b-hmils0) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/containers/GamePage/index.test.js#L32)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

like it? :)
</blockquote></details>

yup :)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

mrniket commented:
Review status: all files reviewed at latest revision, 20 unresolved discussions.

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L7TrHfmo1FmS3ZDQrpV:b53btcr) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
> https://stackoverflow.com/questions/3305869/are-multiple-gitignores-frowned-on
Olaf: both points apply, the javascript project has different files to ignore and the files we all want to ignore for this project are in the subdirectory game_frontend. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

mrniket commented:
Review status: 27 of 32 files reviewed at latest revision, 20 unresolved discussions.

---

*[README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70DZ1mYtAEuouqVDuW:-L7U86zSv5BU6hw7HlE5:b-896fix) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L19)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

`**Note**: The` would look better
</blockquote></details>

Done.

---

*[README.md, line 23 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70eOFWYlpEVDb48Oow:-L7U89-vaHGVfcM_qk22:b-896fix) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L23)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

:thumbsup: I think I'm going to rewrite this section, I'll use the comments here to guide me
</blockquote></details>

Done.

---

*[game_frontend/README.md, line 7 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714B5UX0-sNDq0ccXf:-L7U8185lisQoHxUDUgK:bcn88in) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L7)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

It is written in single page application written in.... => rephrase?
</blockquote></details>

rewritten

---

*[game_frontend/README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714LNxHctgntZ-_kU0:-L7U8CcpbsQJYVJBU_jd:b-896fix) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L19)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

I will remove the `cd` command and just say "go to this folder"
</blockquote></details>

Done.

---

*[game_frontend/README.md, line 61 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L7162EIyVjlqE7mJk_v:-L7U8EYWstTGRkwANiSw:b-896fix) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L61)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

where are the link?
</blockquote></details>

Done.

---

*[game_frontend/README.md, line 67 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L714_trLdqebgIQ6EKO:-L7U8FvhBtHeoZZzTPQP:b-17q4fl) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L67)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

a link to an anchor might be better if possible?
</blockquote></details>

Rewritten.

---

*[game_frontend/README.md, line 77 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L71524AQfhPhkAOiMPt:-L7U8IOXhgw_W2On6HcQ:b5khnp5) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L77)):*
> Rewritten.

---

*[game_frontend/README.md, line 78 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L7156iWABwN2r41_PhK:-L7U8IrM6K5hKZxw-0_t:b-17q4fl) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L78)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

AN onClick?
</blockquote></details>

Rewritten.

---

*[game_frontend/README.md, line 83 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L715EU6l1T1WKDuf2Dp:-L7U8Jf3vn-9378iJJFs:b-17q4fl) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L83)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

features not featutes
</blockquote></details>

Rewritten.

---

*[game_frontend/README.md, line 105 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L715_ynA-5csQC4JYuF:-L7U8KmoC3DdU4SHQMXw:b-17q4fl) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L105)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

this will BE?
</blockquote></details>

Rewritten.

---

*[game_frontend/README.md, line 106 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L71ANgp7_YkFJe99IQ0:-L7U8LMh18SR1O-zjQi5:b-17q4fl) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L106)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

:thumbsup: 
</blockquote></details>

Rewritten.

---

*[game_frontend/README.md, line 113 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L715ir8RGq8DSagF2O6:-L7U8LxJAOjygsezw4Xc:b-17q4fl) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L113)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

create A new state
</blockquote></details>

Rewritten.

---

*[game_frontend/README.md, line 128 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L716A1gbYScE4FLqm7V:-L7U8MV4LBGOqZEa3JoY:b-17q4fl) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/README.md#L128)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

clunky wording?
</blockquote></details>

Rewritten.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Reviewed 6 of 6 files at r4.
Review status: all files reviewed at latest revision, 7 unresolved discussions, some commit checks failed.

---

*[README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70DZ1mYtAEuouqVDuW:-L7UAC5ijoVlXI8VF9H5:bipgpx9) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L19)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

you haven't put the capital letter on The, though, Niket

---

*[README.md, line 23 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70eOFWYlpEVDb48Oow:-L7UAVItHnGlC_SbXi1p:b1m22mm) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L23)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

I can't see the changes or the issue number for this?

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L7UAb0fAN6LXmJj93K-:bvfjrmd) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

> https://stackoverflow.com/questions/3305869/are-multiple-gitignores-frowned-on
Olaf: both points apply, the javascript project has different files to ignore and the files we all want to ignore for this project are in the subdirectory game_frontend. 
</blockquote></details>

What you are putting in this .gitignore are not going to be used in another subdirectory so it looks safe to put it in the main .gitignore. I'm afraid of the confusion / and the inconsistency with our other projects (unless we have local .gitignores elsewhere?)

---

*[game_frontend/src/components/Button/index.js, line 5 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70nDhxoVGOFFgqeAgm:-L7UBX0mLQiy1dhoM5_7:ble80i6) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Button/index.js#L5)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

Yes there is! You can provide a theme. See this for more info: https://www.styled-components.com/docs/advanced#theming
</blockquote></details>

shouldn't we add an example here instead of hard coding the colours? Not a huge deal though to be fair

---

*[game_frontend/src/redux/middlewares/index.js, line 1 at r4](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L7UC2MKcohKYqPSQFf7:-L7UC2MKcohKYqPSQFf8:bb76mu4) ([raw file](https://github.com/ocadotechnology/aimmo/blob/323cb0a8792aa23c9cd666e0fb0e223cfd89357a/game_frontend/src/redux/middlewares/index.js#L1)):*
> ```JavaScript
> 
> ```

Why?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 7 unresolved discussions, some commit checks failed.

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L7UDOnWRXZiM3GnUgyz:b-xzycp2) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

What you are putting in this .gitignore are not going to be used in another subdirectory so it looks safe to put it in the main .gitignore. I'm afraid of the confusion / and the inconsistency with our other projects (unless we have local .gitignores elsewhere?)
</blockquote></details>

In that case you can specify the folder you're ignoring the files in; in the global file, not split them and make devs look for the particular one imo

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

mrniket commented:
Review status: all files reviewed at latest revision, 7 unresolved discussions, some commit checks failed.

---

*[README.md, line 19 at r3](https://beta.reviewable.io/reviews/ocadotechnology/aimmo/500#-L70DZ1mYtAEuouqVDuW:-L7UCxKl0F9SLicf9FCR:b-896fix) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L19)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

you haven't put the capital letter on The, though, Niket
</blockquote></details>

Done.

---

*[README.md, line 23 at r3](https://beta.reviewable.io/reviews/ocadotechnology/aimmo/500#-L70eOFWYlpEVDb48Oow:-L7UD2iU-Z1U2SDHyZup:b-x9z5ro) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L23)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

I can't see the changes or the issue number for this?
</blockquote></details>

https://beta.reviewable.io/reviews/ocadotechnology/aimmo/500 <- that link should help

there is no issue number because it's part of this task

---

*[game_frontend/.gitignore, line 1 at r3](https://beta.reviewable.io/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L7UDHSb92PAIqr2lv9X:b-rl0fra) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

In that case you can specify the folder you're ignoring the files in; in the global file, not split them and make devs look for the particular one imo
</blockquote></details>

I still think it makes sense to separate it but if you guys have a problem with that I'll move it to the top level .gitignore, it's not a huge issue for me

---

*[game_frontend/src/components/Button/index.js, line 5 at r3](https://beta.reviewable.io/reviews/ocadotechnology/aimmo/500#-L70nDhxoVGOFFgqeAgm:-L7ULBLG-EVF-SCz2UOo:bdtqy44) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/src/components/Button/index.js#L5)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

shouldn't we add an example here instead of hard coding the colours? Not a huge deal though to be fair
</blockquote></details>

Yeah I think it might be wise to do it when we do the game IDE so we don't spend too much time doing this

---

*[game_frontend/src/redux/middlewares/index.js, line 1 at r4](https://beta.reviewable.io/reviews/ocadotechnology/aimmo/500#-L7UC2MKcohKYqPSQFf7:-L7ULSjQ8Ie54Z9CKpT0:b-2b8hbd) ([raw file](https://github.com/ocadotechnology/aimmo/blob/323cb0a8792aa23c9cd666e0fb0e223cfd89357a/game_frontend/src/redux/middlewares/index.js#L1)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

Why?
</blockquote></details>

This is there to show the folder structure, you can think of it like the README.md for this folder.

It helps make the imports for the middleware more readable

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: all files reviewed at latest revision, 6 unresolved discussions, some commit checks failed.

---

*[README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70DZ1mYtAEuouqVDuW:-L7ULhzActK6TuGCPJ46:bfhqmln) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L19)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Not done?

---

*[README.md, line 23 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70eOFWYlpEVDb48Oow:-L7ULojmPNIA3k1oiiH9:b2w9bo0) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L23)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

https://beta.reviewable.io/reviews/ocadotechnology/aimmo/500 <- that link should help

there is no issue number because it's part of this task
</blockquote></details>

Hasn't changed for me either

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Review status: 30 of 32 files reviewed at latest revision, 6 unresolved discussions.

---

*[README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70DZ1mYtAEuouqVDuW:-L7UM81XiMf1NzxbNd1i:b-xktxpw) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L19)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

Not done?
</blockquote></details>

Nevermind, got it

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

OlafSzmidt commented:
Reviewed 4 of 6 files at r4, 3 of 3 files at r5.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Reviewed 1 of 3 files at r5.
Review status: all files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[README.md, line 19 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70DZ1mYtAEuouqVDuW:-L7UhgNOWKt9eGUXDUYf:b-9i70f) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L19)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

Nevermind, got it
</blockquote></details>

i see it too thanks

---

*[README.md, line 23 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70eOFWYlpEVDb48Oow:-L7Uhm1z8DOULprvYwRJ:bpehyhe) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L23)):*
<details><summary><i>Previously, OlafSzmidt (Olaf Szmidt) wrote…</i></summary><blockquote>

Hasn't changed for me either
</blockquote></details>

AN isometric?
Also, "the bigger the world gets" should be working by April but it is not doing that now i think

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

mrniket commented:
Review status: all files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

*[README.md, line 23 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70eOFWYlpEVDb48Oow:-L7Up0du3oaHzzar5BNP:bmcaamp) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/README.md#L23)):*
<details><summary><i>Previously, CelineBoudier (Celine Boudier) wrote…</i></summary><blockquote>

AN isometric?
Also, "the bigger the world gets" should be working by April but it is not doing that now i think
</blockquote></details>

I think this is being handled in #505 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->

CelineBoudier commented:
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[game_frontend/.gitignore, line 1 at r3](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500#-L70zzYvuonkW2u_4Pa2:-L7UpswDPbsnQAF0xu5d:b-dxt0qz) ([raw file](https://github.com/ocadotechnology/aimmo/blob/ea2b3a935628e177f2a77a17e9e6bf1311aaa065/game_frontend/.gitignore#L1)):*
<details><summary><i>Previously, mrniket (Niket Shah) wrote…</i></summary><blockquote>

I still think it makes sense to separate it but if you guys have a problem with that I'll move it to the top level .gitignore, it's not a huge issue for me
</blockquote></details>

I think both Olaf and I prefer the same file :/

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ocadotechnology/aimmo/500)*
<!-- Sent from Reviewable.io -->
